zombiereloaded/ze_Knife_Stray_v3.cfg

```diff
@@ -1,6 +1,10 @@
+mce_extend 0
+mp_respawn_immunitytime 2
```

Comment:
I warned you this probably wouldn't work well with ZR while we were testing, so I tested this cvar offline and this starts timing from player spawn only, not them becoming infected as a zombie (for both normal & mother infection).